infrastructure/environments/non-prod/main.tf

```diff
@@ -1,13 +1,16 @@
 provider "azurerm" {
-  version = "=1.38.0"
+  version = "~> 1.38.0"
+  # subscription_id = "b2f4966f-71ea-415b-9ec6-bf96d597d596"
```

Comment:
Why is this here? Do we need to regenerate this?

Reply:
Good catch, I removed the comment